portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/usecase/SitetemplatesUsecase.testcase

```diff
@@ -422,19 +424,30 @@ definition {
 			password = "test",
 			userEmailAddress = "userea@liferay.com");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Users",
-			panel = "Control Panel",
-			portlet = "User Groups");
+		Site.openSitesAdmin();
```

Comment:
Line 427 and Line 429 are duplicated check with Line 433. If the new site template doesn't display when adding user group site, then it will not display when adding general site too. So I think it's not necessary to add site template assertion in site admin.

Reply:
Then I'll delete this and the macro too. cc @gaborlovasdotcom